setup.py

```diff
@@ -1,7 +1,16 @@
 from setuptools import setup
 
 setup(
-    name="adb_py",
-    version="0.8",
-    py_modules=["adb_py", "adb_init"],
+    name="adb-pywrapper",
+    version="1.0.0",
+    description="adb-pywrapper facilitates seamless interaction with Android devices using the Android Debug Bridge (ADB) "
```

Comment:
why does this description contain two strings?

Reply:
What do you mean with 2 strings? I only see one

Reply:
Ah yes
"part1"
"part 2"

is the same as
"part1 \
 part2"